evennia/settings_default.py

```diff
@@ -347,6 +347,11 @@
 # default class logs channel messages to a file and allows for /history.
 # This setting allows to override the command class used with your own.
 CHANNEL_COMMAND_CLASS = "evennia.comms.channelhandler.ChannelCommand"
+# These specify defaults to the base Command parent class
```

Comment:
All of these require individual comments. Stuff like `MSG_SHARE` is not intuitive!